.dockerignore

```diff
@@ -1,4 +1,9 @@
-target
-.git
-.github
+target/
```

Comment:
I'm not an expert, but I'm wondering would it work to do it like this:
```shell
# Ignore everything
**

# Only use the binary
!target/release/dali
```

Reply:
You usually want to ship your license and possibly the changelog with your software.